lib/modules/dosomething/dosomething_helpers/dosomething_helpers.share.inc

```diff
@@ -164,3 +164,53 @@ function dosomething_helpers_share_bar($path, $share_types, $share_description,
 
   return $share_bar . "</ul>";
 }
+
+/**
+ * Return social share types and custom content.
+ *
+ * @param object $campaign
+ *   The campaign object to get custom content from.
+ * @return array $social share types
+ *   An array of social share types and custom content.
+ */
+
+function dosomething_helpers_social_share_types($campaign) {
```

Comment:
@sbsmith86 that makes sense to move it there! Just updated it. However, I'm a little confused about the second part. I'm calling it two files - `dosomething_campaign.theme.inc` and `dosomething_signup-.signup_data_form.inc`. Is this wrong to do?